opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/interceptor/HandlerInterceptorSpanDecorator.java

```diff
@@ -132,6 +132,34 @@ public void onAfterConcurrentHandlingStarted(HttpServletRequest httpServletReque
         }
     };
 
+    /**
+     * Decorator to record details about the handler as log events recorded on the span.
```

Comment:
It does not seem to set any logs on the span.

Reply:
thanks, fixed it